### PR TITLE
Normalize styles filter

### PR DESCRIPTION
**Pull request by nickmoreton**
There are different ways that styles are formatted coming out of wordpress.

This mornalizes them so the know what the format is for later parsing.
```
e.g. font-style: italic becomes font-style:italic;
e.g. FONT-WEIGHT:400; becomes font-weight:400;
```
So essentially the styles are all lowercased, with appended ; and have no spaces.

Worth noting that these styles are actually removed later on when the fix_styles
method is run so if they look wrong it's OK here as they are a `template`

Also the mapping for the styles has been reduced to be more manageable.

### CODE REVIEW

File: wagtail_wordpress_import/constants.py [hunk 1]
```diff
@@ -72,176 +75,41 @@
...
-        "font-style: italic; font-size: 85%;",
-        "font-style: italic;font-size:100%;",
-        "font-style: italic;font-size:130%;",
-        "font-style: italic;font-size:85%;",
+        "font-style:italic;",
     ],
     "bold-italic": [
-        "font-style: italic; font-weight: bold;",
-        "font-weight: bold; font-style: italic;",
+        "font-style:italic; font-weight:bold;",
```
Comment (kaedroho) [inline wagtail_wordpress_import/constants.py:85]: What if there's another style inserted in between?

File: wagtail_wordpress_import/functions.py [hunk 2]
```diff
@@ -177,3 +180,32 @@ def clean_pre(m):
         pee = re.sub("(?is)(<pre[^>]*>)(.*?)</pre>", clean_pre, pee)
     pee = re.sub(r"\n</p>$", "</p>", pee)
     return pee
+
+
+def normalize_style_attrs(html):
```
Comment (kaedroho) [inline wagtail_wordpress_import/functions.py:185]: Does it make sense to run this filter without fix_styles, or vice-versa? If not, I think we should merge them into one filter

**Review by kaedroho — APPROVED** [review]
This looks much better now, thanks! Happy for you to address the comments in future PRs to get things unblocked

**Comment by nickmoreton** [comment]
Thanks Karl. I think it might be useful to raise each of the comments here as an issue in github which we can resolve later. That way we can get things merged in. It will help @nimasmi get up to speed I think if he can test out a 'working' package. What do you think? and perhaps for the other PR's too. Then Im thinking adding an issue/s for the discussions we had earlier today?

**Comment by kaedroho** [comment]
Yep, feel free to merge this one in its current state, it's already a massive improvement. I think we can resolve these in separate PRs

File: wagtail_wordpress_import/constants.py (hunk 1 above)
Comment (nickmoreton) [reply wagtail_wordpress_import/constants.py:85]: #40

File: wagtail_wordpress_import/functions.py (hunk 2 above)
Comment (nickmoreton) [reply wagtail_wordpress_import/functions.py:185]: #41

**Pull request merged**